handlers/character.py

```diff
@@ -205,25 +207,33 @@ def get_status(other_username, db, chat_id, username):
     character = get_linked_character(db, chat_id, search_param)
 
     return (f'{character.name} | {character.race} {character._class} Level {character.level}\r\n'
-            f'HP: {character.current_hit_points}/{character.max_hit_points} | XP: {character.current_experience}')
+            f'HP: {character.current_hit_points}/{character.max_hit_points} | XP: {character.current_experience}/{character.experience_needed}')
+
+def get_currency(other_username, db, chat_id, username):
```

Comment:
Isn't better to have a `/set_currency <username> <currency>` command (for DMs) that receives the username and the currency in format `<number><unit>`. For example:
`/set_currency @satanas 35gp`